### ISSUE TITLE
question about alpha

### ISSUE THREAD

**Issue by quxue** [ElementAI/TADAM#7]
Alpha is learnable in the paper, but inside the code
"`Flags.metric_multiplier_trainable=False`" means that alpha is a fixed value. Could you tell me why?

**Comment by boreshkinai** [comment]
This is because alpha is cross-validated on the validation set. There has been other configurations in which alpha was learnable, that included the polynomial with learnable coefficients of the metric fed into softmax.
